### PR TITLE
Fix benchmarks

### PR DESCRIPTION
**Pull request by 4e6**
[ci no changelog needed]

### Checklist

Please include the following checklist in your PR:

- [x] The documentation has been updated if necessary.
- [x] All code conforms to the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides.
- All code has been tested:
  - [x] Unit tests have been written where possible.
  - [ ] ~If GUI codebase was changed: Enso GUI was tested when built using BOTH
        `./run ide dist` and `./run ide watch`.~

### CODE REVIEW

File: engine/runtime/src/bench/scala/org/enso/interpreter/bench/fixtures/semantic/AtomFixtures.scala [hunk 1]
```diff
@@ -28,18 +28,18 @@ class AtomFixtures extends DefaultInterpreterRunner {
   val generateListQualifiedCode =
     """from Standard.Base.Data.List import all
       |
-      |main = length ->
-      |    generator = acc -> i -> if i == 0 then acc else @Tail_Call generator (List.cons i acc) (i - 1)
+      |main self = length ->
+      |    generator = acc -> i -> if i == 0 then acc else @Tail_Call generator (Cons i acc) (i - 1)
```
Comment (kustosz) [inline engine/runtime/src/bench/scala/org/enso/interpreter/bench/fixtures/semantic/AtomFixtures.scala]: Should be List.Cons – the point of this benchmark is to verify there's no overhead in the qualified form.

**Review by kustosz — APPROVED** [review]
Awesome, just one change I'd do differently, otherwise we have 2 identical benchmarks

**Review by JaroslavTulach — APPROVED** [review]

**Comment by JaroslavTulach** [comment]
Once this PR is merged, we can also integrate the #3640 gate check.

**Review by hubertp — CHANGES_REQUESTED** [review]
So `self` in main is actually not needed, just need to change the way we invoke it. I will change it

**Review by hubertp — APPROVED** [review]

**Pull request merged**